planning/behavior_path_planner/src/utils/goal_planner/freespace_pull_over.cpp

```diff
@@ -114,17 +114,20 @@ boost::optional<PullOverPath> FreespacePullOver::plan(const Pose & goal_pose)
   }
 
   utils::correctDividedPathVelocity(partial_paths);
+  std::vector<std::pair<double, double>> pairs_terminal_velocity_and_accel{};
+  pairs_terminal_velocity_and_accel.push_back(std::make_pair(velocity_, 0));
+  pairs_terminal_velocity_and_accel.push_back(std::make_pair(-velocity_, 0));
```

Comment:
why set `-velocity_` and assume parttial.paths.size()=2?

Reply:
thanks!!

Reply:
I define in https://github.com/autowarefoundation/autoware.universe/pull/4804/commits/172eac02af271dc5ace9a7455340b16e0ccda968